src/README.md

```diff
@@ -18,18 +18,20 @@ As a Platform as a Service, or PaaS, Platform.sh automatically manages everythin
 
 Platform.sh covers not only all of your hosting needs but also most of your DevOps needs. It is a simple, single tool that covers the application life-cycle from development to production and scaling.
 
-You only need to write your code, including a few YAML files that specify your desired infrastructure, commit it to Git, and push.  You don't need to setup anything manually. The web server is already setup and configured, as are any database, search engine or cache that you specify.
+You only need to write your code, including a few YAML files that specify your desired infrastructure, commit it to Git, and push.  You don't need to setup anything manually. The web server is already setup and configured, as is any database, search engine or cache that you specify.
```

Comment:
This one's subtle. It's a list of items (plural), but the items are singular.  So in aggregate is that singular or plural?  Is this a US/UK divide? :smile:

Reply:
It's the "or." If it were "and" then it would be "are," but "or" gets "is." Also, there's a missing oxford comma I forgot to add.

Reply:
I'll go with that.  And Oxford Comma Forevar!